core/domain/activity_services_test.py

```diff
@@ -219,9 +234,11 @@ def test_split_by_type(self):
                 exploration_123, collection_def, exploration_ab]),
             (['123', 'ab'], ['def']))
 
-    def test_split_by_type_raises_error_if_given_invalid_activity_ref(self):
+    def test_split_by_type_raises_error_if_given_invalid_activity_ref(
+        self
+    ) -> None:
         exploration_123 = self._create_exploration_reference('123')
-        with self.assertRaisesRegex(Exception, 'Invalid activity reference'):
+        with self.assertRaisesRegex(Exception, 'Invalid activity reference'): # type: ignore[no-untyped-call]
```

Comment:
When would the no-untyped-call statements be resolved?

Reply:
I think these files should be covered in following manner:
```
core/domain (because this is the folder which contain most annotated files.)
   |
tests/ (because this folder contains most general purpose packages/modules. like utils.py, constant.py, feconf.py, test_utils.py and etc )
   |
core/controller (because this is Imp to cover.)
   |
   .
   .
```

Reply:
@nithusha21 PTAL!